### PR TITLE
[UT03-981] [UT03-982] bugfix/Additional fixes + sentry exception

### PR DESCRIPTION
**Pull request by TiborPL**
# Description

- check proper team membership before calling the changelog (include country focal point)
- check numberic fields by values and null instead of just falsy
- add currency field
- do not show N/A “changes”
- do not pass null to date formatter, return empty string instead
- parse CPD field
- for  non-team members, check if project is published and show the published version as current version in the header

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/components/dialogs/ProjectHistoryDialog.vue [hunk 1]
```diff
@@ -467,6 +498,10 @@ export default {
       const country = this.getCountryDetails(countryId)
       return country && country.name ? country.name : ''
     },
+    parseCountryOffice(countryId) {
+      const country = this.getCountryDetails(countryId)
+      return country && country.name ? country.name : ''
```
Comment (nullcode) [inline frontend/components/dialogs/ProjectHistoryDialog.vue]: why is the country office taking the countryId (same as in parseCountry), they are different objects

File: frontend/components/dialogs/ProjectHistoryDialog.vue [hunk 2]
```diff
@@ -491,6 +526,15 @@ export default {
       }
       return ''
     },
+    parseCustomAnswers(answers) {
```
Comment (nullcode) [inline frontend/components/dialogs/ProjectHistoryDialog.vue:532]: does this work for both donor custom and country custom answers?

File: frontend/components/project/history/values/ValueText.vue [hunk 3]
```diff
@@ -30,6 +24,11 @@ export default {
       required: true,
     },
   },
+  methods: {
+    isNothing(value) {
+      return value === null || value === ''
```
Comment (nullcode) [inline frontend/components/project/history/values/ValueText.vue:29]: perhaps should contain empty lists [] ?

**Review by nullcode — CHANGES_REQUESTED** [review]

File: frontend/components/project/history/values/ValueText.vue (hunk 3 above)
Comment (TiborPL) [reply frontend/components/project/history/values/ValueText.vue:29]: ValueText component only receive simple texts or null, ValueTags is handling lists..

File: frontend/components/common/ProjectBar.vue [hunk 4]
```diff
@@ -147,10 +145,10 @@ export default {
...
     },
     isTeam() {
-      if (this.user) {
-        return this.user.member.includes(+this.$route.params.id)
-      }
-      return false
+      return this.user
+        ? this.user.member.includes(+this.$route.params.id) ||
+            this.user.manager_of.includes(this.project.country_office)
+        : false
```
Comment (GergelyKovach) [inline frontend/components/common/ProjectBar.vue:151]: i am a huge fan of ternary 👏, but this makes the whole code less human-readable

Comment (TiborPL) [reply frontend/components/common/ProjectBar.vue:151]: Well, as long as we have the linter and prettier linked and forced, it is what it is :)

File: frontend/components/dialogs/ProjectHistoryDialog.vue (hunk 2 above)
Comment (TiborPL) [reply frontend/components/dialogs/ProjectHistoryDialog.vue:532]: No, `donor_answers` and `country_answers` are ignored (just like in export/import functions)

File: frontend/components/dialogs/ProjectHistoryDialog.vue (hunk 1 above)
Comment (TiborPL) [reply frontend/components/dialogs/ProjectHistoryDialog.vue]: Fixed

**Review by nullcode — APPROVED** [review]

**Pull request merged**